_data/people.yaml

```diff
@@ -3448,7 +3443,6 @@ lpantano:
   last-name: Pantano
   pronouns: she/her
   twitter: lopantano
-  website: https://lpantano.github.com
```

Comment:
To replace by https://lpantano.github.com

Reply:
@bebatut this GitHub link isn't working

Reply:
Sorry: https://lpantano.github.io/